src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Plane plane = new Plane.PlaneBuilder().setLength(37.6).setHeight(11.8).setMaximumSpeed(890)
+                .setBrand("Airbus").setModel("A320").setEmptyWeight(42100).build();
```

Comment:
this looks more readable in case of a pipeline (when we are calling multiple methods returning the same object in a row)
```suggestion
        Plane plane = new Plane.PlaneBuilder()
                .setLength(37.6)
                .setHeight(11.8)
                .setMaximumSpeed(890)
                .setBrand("Airbus")
                .setModel("A320")
                .setEmptyWeight(42100)
                .build();
```

Reply:
Done